db/schema.yml

```diff
@@ -1557,13 +1557,15 @@ jobs:
 - target_id
 - target_class
 - type
+- process
```

Comment:
New columns should not insert themselves int he middle of the row, otherwise it will break replication.  This *should* fail one of the existing tests that look for this.